ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -468,14 +471,11 @@ export class AccountCreationInstitution extends React.Component<
                       data-test-id='role-dropdown'
                       style={styles.wideInputSize}
                       placeholder={
-                        getRoleOptions(institutions, institutionShortName)
-                          ? ''
+                        institutionShortName
```

Comment:
Does this placeholder still make sense? It seems like institution should no longer have an effect on roles.